src/lib/web/LocaleStrings.js

```diff
@@ -20,6 +21,9 @@ const LocaleStrings = function (inputKey, reqLocale) {
     return localeStrings[inputKey]
   } else if (defaultLang[inputKey] !== undefined) {
     return defaultLang[inputKey]
+  } else if (unusedDefaultLang[inputKey] !== undefined) {
+    console.warn(`Please move string "${inputKey}" from "default.unused.js" to "default.js"`)
+    return unusedDefaultLang[inputKey]
```

Comment:
Fallback. I use it cause defining unused strings I was doing during application runtime.  So some of used strings could be left in `default.unused.js`.